src/deepforest/model.py

```diff
@@ -209,12 +253,20 @@ def write_crops(self, root_dir, images, boxes, labels, savedir):
 
         # Use rasterio to read the image
         for index, box in enumerate(boxes):
-            xmin, ymin, xmax, ymax = box
             label = labels[index]
             image = images[index]
             with rasterio.open(os.path.join(root_dir, image)) as src:
-                # Crop the image using the bounding box coordinates
-                img = src.read(window=((ymin, ymax), (xmin, xmax)))
+                # Get image dimensions
+                image_width = src.width
+                image_height = src.height
+                
+                # Expand the bounding box to a square
+                xmin, ymin, xmax, ymax = box
+                square_box = self.expand_bbox_to_square([xmin, ymin, xmax, ymax], image_width, image_height)
```

Comment:
According to the docs `boxes` it looks like boxes is a nested list in which case we wouldn't need to split it into individual objects and then reassemble it. So these two lines could be come:

```python
square_box = self.expand_bbox_to_square(box, image_width, image_height)
```

But I haven't explicitly checked.